Library/test-helpers/lib.sh

```diff
@@ -1715,6 +1715,254 @@ limeSubmitCommonLogs() {
 
 }
 
+true <<'=cut'
+=pod
+
+=head2 limeconCreateNetwork
+
+Create container network.
+
+    limeconCreateNetwork
+
+=over
+
+=item NETWORK_IP
+
+IP of container network.
+
+=item NAME_NETWORK
+
+Name of container network.
+
+=back
+
+Returns 0.
+
+=cut
+
+limeconCreateNetwork() {
+
+    local NETWORK_IP=$1
+    local NAME_NETWORK=$2
+
+    if [ -z "${NAME_NETWORK}" ] || [ -z "${NETWORK_IP}" ]; then
+        echo "Name of network or network IP is not specified!"
+        return
+    fi
+
+    podman network create --subnet=$NETWORK_IP $NAME_NETWORK
+    echo "Network name: ${NAME_NETWORK} , ip:${NETWORK_IP}"
+
+}
+
+true <<'=cut'
+=pod
+
+=head2 limeconDeleteNetwork
+
+Delete container network.
+
+    limeconDeleteNetwork
+
+=over
+
+=item NAME_NETWORK
+
+Name of the container network.
+
+=back
+
+Returns 0.
+
+=cut
+
+limeconDeleteNetwork() {
+
+    local NAME_NETWORK=$1
+
+    podman network inspect $NAME_NETWORK
+
+}
+
+true <<'=cut'
+=pod
+
+=head2 limeconPrepareAgentImage
+
+Prepare podman image with keylime agent. Add dnf repos into podman image, install keylime agent and
+copy CA cert from the host into image and set needed permissions.
+
+    limeconPrepareAgentImage
+
+=over
+
+=item NAME_TAG
+
+Name of image tag, if paramer empty name tag set as agent_image.
+
+=back
+
+Returns 0.
+
+=cut
+
+limeconPrepareAgentImage() {
+
+    local IMAGE=$1
+    local NAME_TAG=$2
+
+    #copy cacert to work dir for podman build
+    cp /var/lib/keylime/cv_ca/cacert.crt .
+    
+    if podman images | grep agent_images; then
```

Comment:
I am not sure about this one.. I would probably remove it. `agent_images` should not match anyway since your default is different and there can be custom tag too.

Reply:
That is typo. So do you suggest a use and grep different tag, something like keylime_agent_images or just remove whole conditional?

Reply:
remove it completely.